rekcurd_dashboard/apis/api_evaluation.py

```diff
@@ -109,6 +151,29 @@ def delete(self, project_id: int, application_id: str, evaluation_id: int):
         return {"status": True, "message": "Success."}
 
 
+@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluations/<int:evaluation_id>/download')
+class ApiEvaluationIdDownload(Resource):
+    def get(self, project_id: int, application_id: str, evaluation_id: int):
```

Comment:
@yuki-mt 
S3 and GCS require user authentication when we download something from them. But if our user **DON'T** set a LDAP authentication mode on Rekcurd dashboard, it means dashboard will be a security hole to them because S3/GCS cannot know who request a download correctly.

But...
It is difficult to restrict the download permission at the moment in our architecture. Your current implementation may be OK right now. How do you think?

(Please forget about my saying of `post` request. I just worry about that web crawlers will access `GET` endpoint and will download files. I am thinking it will be a risk for users.)

Reply:
👍